hazelcast/src/main/java/com/hazelcast/util/StringUtil.java

```diff
@@ -288,4 +291,66 @@ public static String getterIntoProperty(String getterName) {
         }
         return propertyName;
     }
+
+    /**
+     * Trim whitespaces. This method (compared to {@link String#trim()}) doesn't limit to space character.
+     *
+     * @param input string to trim
+     * @return null if provided value was null, input with removed leading and trailing whitespaces.
```

Comment:
You could use `{@code null}` in all JavaDoc of this class. I also think that `@return` docs should have no trailing period, at least this is how I reformatted all recent JavaDocs.